inference-engine/thirdparty/clDNN/src/CMakeLists.txt

```diff
@@ -140,6 +140,7 @@ include_directories(
     "${__CLDNN_Directory__ks_core}/common"
     "${__CLDNN_Directory__ks_actual_kernels}"
     "${__CLDNN_Directory__ks_common}"
+    "${IE_MAIN_SOURCE_DIR}/src/plugin_api/"
```

Comment:
Is it really needed?

Reply:
They are needed to include some os relevant files. We could copy them too, but it needs too many copies. How do you think ?